TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.m

```diff
@@ -73,35 +73,84 @@
 #import "TSDKEventLineup.h"
 #import "TSDKEventLineupEntry.h"
 
-static NSArray *supportedSDKObjects;
+static NSArray *_supportedSDKObjects;
 static NSArray *knownCompletionTypes;
 
 @implementation TSDKObjectsRequest
 
 + (NSArray *)supportedSDKObjects {
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        NSMutableArray *supportedObjects = [[NSMutableArray alloc] init];
-        
-        unsigned int classCount = 0;
-        Class *classList = objc_copyClassList(&classCount);
-        
-        for (unsigned int i = 0; i < classCount; i++) {
-            NSString *className = [NSString stringWithUTF8String:class_getName(classList[i])];
-            if([className hasPrefix:@"TSDK"]) {
-                Class class = NSClassFromString(className);
-                
-                if(class && [class isSubclassOfClass:[TSDKCollectionObject class]]) {
-                    [supportedObjects addObject:class];
-                }
-            }
-        }
-        if(classList != NULL) {
-            free(classList);
-        }
-        supportedSDKObjects = [supportedObjects copy];
+        _supportedSDKObjects = @[[TSDKTeamFee class],
```

Comment:
What is this list used for and when should we be updating it from now on?

Reply:
this list is used when getting an array of Collection+JSON objects from the server, and converting them into ObjC classes. 

In the instructors for adding classes to the SDK there is a comment you need to import the file into the header for this file. That caused the list to be generated in real time. For the right now, you will need to also add the type to this list.

If this reduces the crashes we should start generating the list at the CI level.

Reply:
If that's the intention, having this list will not make a difference in the contents of the list. `objc_copyClassList` will give you a comprehensive list of all classes loaded into the runtime regardless of if you have imported the header into this class or not. We can import only what we need to compile this class and it will provide the same classes in `objc_copyClassList`.

Reply:
I'm saying this change to this file might not be required at all. I just executed this to verify and `objc_copyClassList` listed GoogleAds, Lottie, Firebase, underscored, and private Apple classes. Everything that I anticipated.

Reply:
Feel free to close the PR if it's mot needed. I'll take a deeper look another time.

Reply:
@larsacus check out `+ (TSDKCollectionObject *)teamSnapObjectFromCollectionJSON:(TSDKCollectionJSON *)collectionJSON` to see usage of this array.

Reply:
ah I see that now. yeah seems pretty fundamental.